udata/core/dataset/search.py

```diff
@@ -168,7 +170,16 @@ def is_indexable(cls, dataset):
                 not dataset.private)
 
     @classmethod
+    def get_suggest_weight(cls, temporal_weight, spatial_weight, featured):
+        '''Compute the suggest part of the indexation payload'''
+        featured_weight = 1 if not featured else 2
+        # XXX temporal_weight seems too high
```

Comment:
@noirbizarre cued to cap `temporal_weight` to `2`.

s/temporal_weight */max(temporal_weight, 2) */